### ISSUE TITLE
Can't find GraphQLNode

### ISSUE THREAD

**Issue by edopelawi** [justinmakaila/Swift-GraphQL#5]
Hi,
I tried to implement GraphQLNode like on the README, but Xcode doesn't detect it. I tried to install it using Carthage for version 0.2.3. 

Is the README.md updated? Thank you! 😄

**Comment by justinmakaila** [comment]
No, it's not. The interface changed a little bit. Please check out the `GraphQL.swift` file. I'll update the `README` eventually, but I'm happily accepting PRs 👍
